src/NUnitConsole/nunit3-console/TestEventHandler.cs

```diff
@@ -42,6 +42,9 @@ public class TestEventHandler : MarshalByRefObject, ITestEventListener
         private readonly bool _displayAfterTest;
         private readonly bool _displayBeforeOutput;
 
+        private string _currentTestName;
```

Comment:
_lastTestOutput ? or something like that? IOW, we use "current" to mean the label under which we are producing output in _currentLabel, so _currentTestName seems a tad confusing.